houdini/scripts/AfterLastDelete.py

```diff
@@ -0,0 +1,12 @@
+def run_after_last_delete(node):
+    try:
+        import sgtk
+    except ImportError:
+        pass
+    engine = sgtk.platform.current_engine()
+    handler = engine.node_handler(node)
+    if handler:
+        handler.after_last_delete(node=node)
```

Comment:
If you really want to play it safe in case the `engine` is not initialised:
```suggestion
    if engine:
        handler = engine.node_handler(node)
        if handler:
            handler.after_last_delete(node=node)
```

Again, same suggestion for all other `houdini/scripts/*.py`

Fun fact! If this was Python 3.8 only, I'd suggest this instead

```python
    if engine := sgtk.platform.current_engine():
        if handler := engine.node_handler(node):
            handler.after_last_delete(node=node)
```